content/articles/eof.md

```diff
@@ -17,9 +17,9 @@ This is my final post on this blog as a Columbia employee. Taking a cue from our
  
 I started working at the Columbia Center for New Media Teaching and Learning (CCNMTL) in May 2004. While I’ve probably shared the story of how I landed at Columbia hundreds of times, I’ve never captured it fully in writing. I’ve written extensively about the circumstances of my departure from my previous employer for a course on the Political Economy of Media at the New School back in 2007 [_Fabricating Freedom: Free Software Developers at Work and Play_](http://pocketknowledge.tc.columbia.edu/home.php/viewfile/38498), but that essay focused more on an analysis of free software and labor, and neglected to explore the connections between open source communities and education.
 
-As I wrote in _Fabricating Freedom_, at the beginning of the millennium I found myself backpacking across Eastern Europe as a software developer, participating in coding hackathons held in Viennese castles and Norwegian fjords. Intermingling with artists and social theorists was an essential component of these gatherings. I wasn’t particularly politically conscious as an undergrad, and my experiences within a vibrant open sources community awakened my interest in what some might call politics — governance, decision making and conflict resolution. I also came to understand the Plone community as a self-organized knowledge community, one that continually promotes active learning among all participants.  I would’ve not read Seymour Papert’s  [_Mindstorms_](https://en.wikipedia.org/wiki/Mindstorms_%28book%29) until years later, but looking back it is easy to recognize how this open source community began to fulfill Papert’s vision of schools without walls or classrooms. I also experienced the lessons of Jacques Rancière’s [_The Ignorant Schoolmaster_](https://en.wikipedia.org/wiki/Jacques_Ranci%C3%A8re#Life_and_work Ignorant Schoolmaster) first-hand as the Python community collectively embarked on intellectual emancipation.
```

Comment:
I want to note that these long lines make these diffs very hard to read. When I write hugo posts I limit the line length in the markdown files to 80 characters.